File: 6_Average/6.py
def leerElementos():
    print("Escribe los elementos: ")
    return list(map(int, input().split()))

def calcularPromedio(a):
    prom = 0
    for i in range(len(a)):
        prom = prom + a[i]
    return prom / len(a)

a = leerElementos()
print("El promedio es: " + str(calcularPromedio(a)) + "\n")

File: 6_Average/6.rb
def leerElementos
    puts "Escribe los elementos: "
    return gets.scan(/\d+/).map(&:to_i)
end

def calcularPromedio(a)
    prom = 0.0
    a.each do |i|
        prom += i
    end
    return prom / a.length
end

a = leerElementos
print "El promedio es: #{calcularPromedio(a).round(3)} \n"
